dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.37/V2_37_22__Create_table_for_program_stage_instance_audit.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE programstageinstanceaudit (
```

Comment:
It is a good practice to make all the scripts idempotent.
`CREATE TABLE IF EXISTS` will do the trick

Reply:
resolved